client/src/Questions/QuestionForm.js

```diff
@@ -9,9 +9,12 @@ import ValidationError from '../ValidationError';
 
 function QuestionForm() {
   const { id } = useParams();
+  const query = new URLSearchParams(useLocation().search);
```

Comment:
When we're creating a new Question, we pass in the initial QuestionnaireId we wish to create the Question for... this is how we extract a query param (i.e. from a url path that looks like `/questions/new?QuestionnaireId=[id]`...